proof/proofd/src/XrdProofdProofServMgr.cxx

```diff
@@ -4642,10 +4642,10 @@ bool XrdProofdProofServMgr::Alive(XrdProofdProtocol* p)
       int rect = now - iter->second;
       if (rect < fReconnectTimeOut) {
          if (p == iter->first) alive = false;
+         ++iter;
       } else {
-         fDestroyTimes.erase(iter);
+         iter = fDestroyTimes.erase(iter);
       }
-      iter++;
```

Comment:
@gganis Hi Gerri, could you please comment here? I'd like to merge these fixes, but I'm unsure about this particular change. Thanks!